libs/ledger-live-common/src/bot/index.ts

```diff
@@ -249,6 +249,15 @@ export async function bot({
   const { GITHUB_RUN_ID, GITHUB_WORKFLOW } = process.env;
 
   let body = "";
+  let githubBody = "";
+  function appendBody(content) {
+    body += content;
+    githubBody += content;
+  }
+  function appendBodyFullOnly(content) {
```

Comment:
Just a suggestion but maybe we could use only one method and use a boolean as input

Reply:
I started doing this at first and then it was making the code using it totally unreadable, with a floating "true" which makes it also easier to do mistake :)